alchemy-web/providers/neon/database.md

```diff
@@ -0,0 +1,49 @@
+# Neon Database
+
+A Neon database is a PostgreSQL database within a branch. Each branch can contain multiple databases.
+
+## Properties
+
+- `project_id` (string, required): The ID of the Neon project
+- `branch_id` (string, required): The ID of the branch where the database will be created
+- `name` (string, required): The name of the database
+- `owner_name` (string, required): The name of the role that will own the database
+- `adopt` (boolean, optional): Whether to adopt an existing database if it already exists. Default: false
+
+## Dependencies
+
+- Requires a Neon Project to exist
+- Requires a Neon Branch to exist
+- Requires a Neon Role to exist (specified as owner_name)
```

Comment:
Don't bother documenting the Properties or Dependencies in any of the docs, it's noise. Purely focus on usage-based docs. Take a look at our other provider docs to get the idea.